common/country_types/hfe_country_types_pirate.txt

```diff
@@ -173,45 +180,60 @@ hfe_pirate = {
 		category = country_base
 		produces = {
 			energy = 50
-			minerals = 20
-			food = 10
+			minerals = 50
+			food = 50
```

Comment:
I think pirate cheat resources should be moved from country base and modifier to origin.
So player can see it.
Also it should be iasier to balance all cheats in one place.

Reply:
The player doesn't need to see it.

Reply:
Pirate do not need 50 food anyway

Reply:
Pirates have no other way to get Food.

Reply:
They have

Reply:
Name it.

Reply:
Jobs obviously, with slave job every pirate will have good surplus of food without cheats.

Reply:
Okay, done